src/Adapter/PlatformServices.Shared/netstandard1.3/Services/ns13TestContextPropertyStrings.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices
+{
+#pragma warning disable SA1649 // File name must match first type name
+
+    internal static class TestContextPropertyStrings
+    {
+        /// <summary>
```

Comment:
Move this comment before class declaration

Reply:
done